### PR TITLE
Changes to the grammar to support the latest isla

### PR DESCRIPTION
**Pull request by ric-almeida**
Changes to the grammar to support parsing of .isla traces and generating the corresponding .v traces using the new syntax for enumerate types and values.

### CODE REVIEW

**Comment by ric-almeida** [comment]
@Mjiig I was going to add you as a Reviewer but that option isn't accessible to me. Are you able to review this?

**Comment by Mjiig** [comment]
Oh also, unless I'm missing something the committed .v file still needs updating?

File: isla_lang.ott [hunk 1]
```diff
@@ -366,7 +341,7 @@ smt :: '' ::= {{auxparam 'a }}
      % (define-enum n) is a sugar for
      % (declare-datatypes () ((Enum_n en_0 en_1 ... en_(n-1) ))) but
      % it can appear multiple times and the type must be declared only once
```
[the remark this answers is not in the document]
Comment (ric-almeida) [reply isla_lang.ott]: Updated: 
```
    % ( define-enum name int ( name1 .. namej ) ) defines a new enumerate type called `name`
    % containing `int` different constructors: `name1`, ..., `namej`.
```

**Comment by Mjiig** [comment]
Maybe? I think it was originally being used to make getting at the .v file from opam easier, I guess something's changed since you have islaris building without it, but I don't know exactly what it was being used for so I'm hesitant to remove it. I also think being able to look at the Coq types in the checked in repo is sometimes useful, if only because I find Coq easier to read than ott.

**Comment by ric-almeida** [comment]
> Maybe? I think it was originally being used to make getting at the .v file from opam easier, I guess something's changed since you have islaris building without it, but I don't know exactly what it was being used for so I'm hesitant to remove it. I also think being able to look at the Coq types in the checked in repo is sometimes useful, if only because I find Coq easier to read than ott.

That makes sense, but to prevent it from becoming a misleading source of information, I've updated the comment at the head of the file:
```
(* Generated by Ott 0.33 from: isla_lang.ott 
   To keep it consistent with isla_lang.ott, update this file manually after a new build. *)
```

**Pull request merged**